### ISSUE TITLE
travis-ci

### ISSUE THREAD

**Issue by dynamitechetan** [TheAlgorithms/Python#12]
@H-Darji why you removed python version 2.7 from travis.yml file? Many people are still using python ver 2.7.

**Comment by H-Darji** [comment]
I removed it from  .travis.yml file, because it fails build when we push or merge (reason is some of the python scripts are written in python >b3 and dose not support python < 3). It does not affect actual code, when I make all the scripts to run in all python versions, I'll add 2.7 back to .travis.yml.

**Comment by AnupKumarPanwar** [comment]
Yes even I think that some merge requests are in different versions  and merging such requests may lead to spoiling of whole code.. And we will land back to the issue of non-pythonic code
